src/lex-keyword.cpp

```diff
@@ -372,7 +375,7 @@ const struct keyword_entry *lexer_keyword::look_up(const char *str,
       if (o >= 0) {
         const char *s = o + stringpool;
 
-        if (*str == *s && !strncmp(str + 1, s + 1, len - 1) && s[len] == '\0')
+        if (*str == *s && !strncmp(str + 1, s + 1, len - 1) && s[len] == ' ')
```

Comment:
Must fix: You introduced a syntax error.

Reply:
I believe it was caused by clang-format. downgrading to v9 solved the issue